opentelemetry-instrumentation/tests/test_auto_tracing.py

```diff
@@ -104,3 +108,24 @@ def test_trace_init_otlp(self):
             "my-otlp-test-service",
         )
         del environ["OTEL_SERVICE_NAME"]
+
+    @patch.dict(
+        environ, {"OTEL_SERVICE_NAME": "my-random-ids-generator-test-service"}
+    )
+    def test_trace_init_default_random_ids_generator(self):
```

Comment:
is it worth testing the environment variable configuring the ids_generator? 

If you don't, there's a risk that someone may modify that code to not work, and we'd have no way of validating programmatically.

Reply:
Yes it is worth, I was just unfamiliar with how to do so but I agree it's worth :) I've spent quite some time learning how to do it and so have added because I think it's a useful add! Please let me know what you think!